terraform/e2e/main.tf

```diff
@@ -56,4 +58,5 @@ module "policy_service" {
   pmap_service_account = module.common_infra.run_service_account
   oidc_service_account = module.common_infra.oidc_service_account
   gcs_events_filter    = var.policy_gcs_events_filter
+  pmap_resource_scope  = local.pmap_resource_scope
```

Comment:
Resource scope is not required for policy service.

Reply:
No it's not, but both mapping and policy use the same terraform/pmap-service module.
The resource scope is defined as required in variables.tf, so I have to provide it a value.

Reply:
Forgot to remove?